stdlib/public/core/GroupInfo.json

```diff
@@ -105,7 +105,8 @@
         "DropWhile.swift",
         "PrefixWhile.swift",
         "LazyCollection.swift",
-        "LazySequence.swift"],
+        "LazySequence.swift"
+      ],
```

Comment:
this was updated automatically when I ran `git-clang-format HEAD~1` after modifying this file